spacy/cli/_util.py

```diff
@@ -397,7 +397,11 @@ def git_checkout(
         run_command(cmd, capture=True)
         # We need Path(name) to make sure we also support subdirectories
         try:
-            shutil.copytree(str(tmp_dir / Path(subpath)), str(dest))
+            source_path = tmp_dir / Path(subpath)
+            if not is_subpath_of(tmp_dir, source_path):
+                err = f"{subpath!r} is a path outside of the cloned repository."
```

Comment:
Is this `!r` notation just to put single quotes around the parameter? In all other places in the repo we'd just use `'{par}'` but I guess we can start adopting this notation if it's commonly used - I hadn't seen it before.

(I do think that a small advantage of using the more simple `'{par}'` is that it'll also put `None` inbetween quotes which I think reads more easily)

Reply:
One place I find the `!r` notation very helpful is if it's likely for the value to be a string containing an apostrophe -- like if we needed an error referring to the token "'s", it would automatically switch to double quotes. But that's not this situation, so I'll take out the `!r`.

Reply:
Ah, that's a good point! We might want to consider using it in some other error messages where it could be more relevant :-)